open_academy/i18n/es_MX.po

```diff
@@ -0,0 +1,342 @@
+#: model:ir.model.fields,field_description:open_academy.field_session__id
+msgid "ID"
+msgstr "ID"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__instructor_id
+msgid "Instructor"
+msgstr "Instructor"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_res_partner__is_instructor
+#: model:ir.model.fields,field_description:open_academy.field_res_users__is_instructor
+msgid "Is Instructor"
+msgstr "Es instructor"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.view_partner_form
+msgid "Is Instructor?"
+msgstr "Es un instructor?"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course____last_update
+#: model:ir.model.fields,field_description:open_academy.field_session____last_update
+msgid "Last Modified on"
+msgstr "Ultima modificación"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__write_uid
+#: model:ir.model.fields,field_description:open_academy.field_session__write_uid
+msgid "Last Updated by"
+msgstr "Última actualización por"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__write_date
+#: model:ir.model.fields,field_description:open_academy.field_session__write_date
+msgid "Last Updated on"
+msgstr "Ultima actualización"
+
+#. module: open_academy
+#: model:ir.model.fields.selection,name:open_academy.selection__res_partner__teacher_type__level1
+msgid "Level 1"
+msgstr "Nivel 1"
+
+#. module: open_academy
+#: model:ir.model.fields.selection,name:open_academy.selection__res_partner__teacher_type__level2
+msgid "Level 2"
+msgstr "Nivel 2"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_course__name
+#: model:ir.model.fields,field_description:open_academy.field_session__name
+#: model_terms:ir.ui.view,arch_db:open_academy.session_template
+msgid "Name"
+msgstr "Nombre"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__number_of_attendees
+msgid "Number Of Attendees"
+msgstr "Número de asistentes"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_session__number_of_seats
+msgid "Number Of Seats"
+msgstr "Número de asientos"
+
+#. module: open_academy
+#: code:addons/open_academy/models/session.py:0
+#, python-format
+msgid ""
+"Number of attendees cannot be greater than the number of available seats."
+msgstr "El número de asistentes no puede ser mayor al número de asientos disponibles"
+
+#. module: open_academy
+#: code:addons/open_academy/models/session.py:0
+#, python-format
+msgid "Number of seats must be greater than 0."
+msgstr "EL número de asientos debe ser mayor a 0"
+
+#. module: open_academy
+#: model:ir.ui.menu,name:open_academy.open_academy_menu_root
+msgid "Open Academy"
+msgstr "Academia"
```

Comment:
I know we need a name for our language but i would keep the English name so leave empty the msgtr